### PR TITLE
Refactor: Changes to the Workshop according to sdk changes fromObject() | Constructors

### PR DESCRIPTION
**Pull request by Dudleyneedham**
## fixes https://github.com/KILTprotocol/ticket/issues/172

Updated the Workshop with the recent changes to the SDK

## How to test:

Go through the workshop with the new changes. 
The KILT full-node isn't working completely and everything is working correctly beside that. Known problem.

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

File: claim.md [hunk 1]
```diff
@@ -52,10 +56,13 @@ Append the following code to `2-claim.js`:
 ```javascript
 /* RequestForAttestation needs 3 arguments: 
 * a claim
-* a legitimation (only useful for trust hierarchies, but for this tutorial we stick to a simple case)
 * a claimer
 */
-const requestForAttestation = new Kilt.RequestForAttestation(claim, [], claimer)
+const requestForAttestation = Kilt.RequestForAttestation.fromClaimAndIdentity(
```
Comment (maudnals) [inline claim.md:62]: Suggestion: add `null` as a fourth argument, just for the sake of illustrating the API (see `fromClaimAndIdentity`, it takes 4 arguments)

File: attestation.md [hunk 2]
```diff
@@ -57,7 +61,10 @@ Append the following code to `3-attestation.js`.
 
 ```javascript 
 // build the Attestation object
-const attestation = new Kilt.Attestation(requestForAttestation, attester)
+const attestation = Kilt.Attestation.fromRequestAndPublicIdentity(
+  requestForAttestation,
+  attester
```
Comment (maudnals) [inline attestation.md]: Request:
Change `attester` to `attester.getPublicIdentity()`, to better illustrate that only a `PublicIdentity` is needed here.

**Review by maudnals — CHANGES_REQUESTED** [review]
Looking good! Only two minor things.

File: attestation.md (hunk 2 above)
Comment (Dudleyneedham) [reply attestation.md]: Ok, this is good. I have updated.

File: claim.md (hunk 1 above)
Comment (Dudleyneedham) [reply claim.md:62]: Updated to include Null

**Review by maudnals — APPROVED** [review]
Looks good to me!

**Pull request merged**